### ISSUE TITLE
error showing zram status

### ISSUE THREAD

**Issue by ICHx** [Nefelim4ag/systemd-swap#83]
Hello, I find it unable to show zram-status

I don't understand the code but there might be something not right in line 489
![圖片](https://user-images.githubusercontent.com/11556527/63370449-cb121400-c3b4-11e9-83bb-b235893fa6e6.png)

Thanks for looking at.
```
root# systemd-swap status
Zswap:
.  compressor                 lz4
.  enabled                    Y
.  max_pool_percent           25
.  same_filled_pages_enabled  Y
.  zpool                      z3fold
.  .  duplicate_entry        0
.  .  pool_limit_hit         0
.  .  pool_total_size        0
.  .  reject_alloc_fail      0
.  .  reject_compress_poor   7
.  .  reject_kmemcache_fail  0
.  .  reject_reclaim_fail    0
.  .  same_filled_pages      0
.  .  stored_pages           0
.  .  written_back_pages     0
.  .  compress_ratio         0%
ZRam:
column: invalid option -- '-'
usage: column [-txne] [-c columns] [-s sep] [file ...]
SwapFC:
.        NAME  TYPE  SIZE  USED  PRIO
```

**Comment by Elkropac** [comment]
Hi, if your system is Debian, Ubuntu, it has column utility from bsdmainutils, it has no parameter --table-columns. This version is from util-linux but it's not shipped in util-linux package

**Comment by Elkropac** [comment]
works for me
https://github.com/Elkropac/systemd-swap/commit/fbfdf3c7ab9775274dac6101f6f4507874dc0d66

**Comment by Elkropac** [comment]
I have create pull request for this #86

**Comment by terencode** [comment]
Should be closed as #86 got merged a long time ago.
